CLDSRV-717: Test configured as S3C Integration

BourgoisMickael opened the pull request.

BourgoisMickael committed:
test s3c

BourgoisMickael committed:
tmp disable tst

BourgoisMickael committed:
tmp skip tests for S3C

BourgoisMickael committed:
CLDSRV-717: Fix script ft_route_backbeat

bert-e commented:
# Hello bourgoismickael,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/cloudserver/doc/user).


<details>
  <summary><b>Available options</b></summary>

  name   | description  | privileged   | authored
  ------ | ------------ | ------------ |----------
  `/after_pull_request` | Wait for the given pull request id to be merged before continuing with the current one. |  | 
  `/bypass_author_approval` | Bypass the pull request author's approval |  :star:  | 
  `/bypass_build_status` | Bypass the build and test status |  :star:  | 
  `/bypass_commit_size` | Bypass the check on the size of the changeset ```TBA``` |  :star:  | 
  `/bypass_incompatible_branch` | Bypass the check on the source branch prefix |  :star:  | 
  `/bypass_jira_check` | Bypass the Jira issue check |  :star:  | 
  `/bypass_peer_approval` | Bypass the pull request peers' approval |  :star:  | 
  `/bypass_leader_approval` | Bypass the pull request leaders' approval |  :star:  | 
  `/approve` | Instruct Bert-E that the author has approved the pull request. |  |  :writing_hand: 
  `/create_pull_requests` | Allow the creation of integration pull requests. |  | 
  `/create_integration_branches` | Allow the creation of integration branches. |  | 
  `/no_octopus` | Prevent Wall-E from doing any octopus merge and use multiple consecutive merge instead |  | 
  `/unanimity` | Change review acceptance criteria from `one reviewer at least` to `all reviewers` |  | 
  `/wait` | Instruct Bert-E not to run until further notice. |  | 
  

</details>



<details>
  <summary><b>Available commands</b></summary>

  name   | description  | privileged
  ------ | ------------ | ------------
  `/help` | Print Bert-E's manual in the pull request. | 
  `/status` | Print Bert-E's current status in the pull request ```TBA``` | 
  `/clear` | Remove all comments from Bert-E from the history ```TBA``` | 
  `/retry` | Re-start a fresh build ```TBA``` | 
  `/build` | Re-start a fresh build ```TBA``` | 
  `/force_reset` | Delete integration branches & pull requests, and restart merge process from the beginning. | 
  `/reset` | Try to remove integration branches unless there are commits on them which do not appear on the source branch. | 
  
</details>




*Status report is not available.*

bert-e commented:
# Request integration branches



Waiting for integration branch creation to be requested by the user.

To request integration branches, please comment on this pull request with the following command:

```
/create_integration_branches
```

Alternatively, the `/approve` and `/create_pull_requests` commands will automatically
create the integration branches.

BourgoisMickael committed:
CLDSRV-717: Fix backbeat route parsedContentLength

Because it doesn't go through arsenal's normalizeRequest.
Do the same as for veeam route, to have the log field bytesReceived

BourgoisMickael committed:
CLDSRV-717: Fix tests with location

This file is imported in tests used by S3C that have no
external location configured

BourgoisMickael committed:
CLDSRV-717: Add conditional test skip utils

BourgoisMickael committed:
CLDSRV-717: Unify test routeBackbeat.js

- Reinclude some skip on tests and add new skips for tests not compatible with S3C
- Reinclude versionId encoded in queryObj for backbeat request
- Test backbeat api proxy conditional on config

BourgoisMickael committed:
CLDSRV-717: Unify test routeBackbeatForReplication

- Factorize objMD update account info
- relication scenrio on same account + cross account

BourgoisMickael committed:
fix bucket with replication service user

BourgoisMickael committed:
run backbeat route tests

BourgoisMickael committed:
CLDSRV-717: Rework credentials in backbeat tests

There is no account1 in mem_credentials, we use default
Replace hardcoded credentials so it can run in end to end tests

BourgoisMickael committed:
CLDSRV-717: Dynamic location for backbeat tests

Replace hardcoded us-east-1 with a location from config
So it can run in an end to end tests for S3C

BourgoisMickael committed:
S3C run backbeat test

BourgoisMickael committed:
fixup tests

BourgoisMickael committed:
fixup backbeat test location

BourgoisMickael commented:
This has been split and merged into multiple PR
